src/icon-manifest/supportedExtensions.ts

```diff
@@ -416,6 +416,7 @@ export const extensions: IFileCollection = {
         '.babelrc',
         'babelrc.js',
         '.babelrc.js',
+        '.babelrc.json',
```

Comment:
I've just found `babelrc.json` as a filename without the initial `.`. Is that a typo?